go.mod

```diff
@@ -1,40 +1,42 @@
-module github.com/revel/cmd
+module github.com/bengo4/revel_cmd
```

Comment:
@KoichiWada I suppose this is correct for your fork only, and so this PR is not ok to be merged on revel/cmd